test/integration_tests/python/test_collection.py

```diff
@@ -55,7 +55,6 @@ def teardown_db():
 def create_collection(api):
     # POST - Create a collection
     NEW_COLLECTION_JSON = json.dumps({
-        'curator': 'admin@user.com',
```

Comment:
We don't accept curator as an input field, it is set automatically. This test is probably just out of date.